spec/system/articles/user_edits_an_article_spec.rb

```diff
@@ -4,8 +4,15 @@
   let(:template) { file_fixture("article_published.txt").read }
   let(:user) { create(:user) }
   let(:article) { create(:article, user: user, body_markdown: template) }
+  let(:svg_image) { file_fixture("svg_image.txt").read }
 
   before do
+    SiteConfig.main_social_image = "https://i.ibb.co/NYsy5Fm/JTL-Logo.jpg"
+    SiteConfig.logo_png = "https://i.ibb.co/5nJvG1X/JTL-Logo.png"
```

Comment:
I think these should be called like this:

```ruby
allow(SiteConfig).to receive(: logo_png).and_return("https://i.ibb.co/5nJvG1X/JTL-Logo.png")
```

...This helps _this_ test have the value, but it won't leak into other situations. Technically if you assign `SiteConfig.logo_png = ` directly it will persist between tests and could cause inconsistencies.

Reply:
Thank you! 😄

Reply:
I agree with @benhalpern on the suggestion to use `allow(...).to receive(: ...).and_return()` here! We probably can update the [other](https://github.com/forem/forem/pull/10745/files/712fc183de8030fd3897aaa9b69cfa607b525135#diff-b13a1d18331fa835d78ef96e51770e3bR14) specs in this PR to make sure that we're isolating the exact test where we assign these values :)